config/routes.rb

```diff
@@ -5,6 +5,7 @@
   get "/signup", to: "users#new", as: :signup
   get "/login", to: "sessions#new", as: :login
   get "/home", to: "users#home", as: :home
+  get '/home/security', to: 'users#security', as: :security
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.